contexts/v1.jsonld

```diff
@@ -16,54 +16,39 @@
       }
     },
 
-    "RevocationList2021": {
+    "StatusList2021": {
       "@id":
-        "https://w3id.org/vc/status-list#RevocationList2021",
+        "https://w3id.org/vc/status-list#StatusList2021",
       "@context": {
         "@protected": true,
 
         "id": "@id",
         "type": "@type",
 
+        "statusPurpose":
+          "https://w3id.org/vc/status-list#statusPurpose",
```

Comment:
I'm not sure if this is better than `statusType` ... but maybe. Still considering.

Reply:
I'm concerned about adding too many `xType` fields... people are going to get confused between that and the regular `type` field.

Reply:
Yeah, I think this is probably a good approach.

Reply:
Also agreeing here: `statusPurpose` is a better name than `statusType` which creates ambiguity with numerous of uses of the term `type.`